### DISCUSSION TITLE
Duplicate "[user] requested a deployment" on pull-request

### DISCUSSION THREAD

**Discussion by solarmosaic-kflorence** [community/community#22377] [New to GitHub]
<p>We have GitHub workflows with jobs that are configured to run against an environment (e.g. <code>environment: Preview</code>). When these workflows run, GitHub updates the pull-request UI to denote that the user requested a deployment to the environment. This was working fine until recently, it looks like the UI is now adding these updates twice:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/2/02e9be42ba312739b2d644e14dab52e7a06550e4.png" data-download-href="/uploads/short-url/pLKlc7XLtEr8NFdgTtaD6GwGna.png?dl=1" title="Screen Shot 2021-11-04 at 11.35.25 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/52473140/181070650-78866ce0-1038-41ca-a981-ba4a8472769c.png" alt="Screen Shot 2021-11-04 at 11.35.25 AM" data-base62-sha1="pLKlc7XLtEr8NFdgTtaD6GwGna" width="690" height="383" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/2/02e9be42ba312739b2d644e14dab52e7a06550e4_2_690x383.png, https://aws1.discourse-cdn.com/github/original/3X/0/2/02e9be42ba312739b2d644e14dab52e7a06550e4.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/2/02e9be42ba312739b2d644e14dab52e7a06550e4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/2/02e9be42ba312739b2d644e14dab52e7a06550e4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-11-04 at 11.35.25 AM</span><span class="informations">702×390 46.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The issue goes away when refreshing the page:</p>
<p><img src="https://user-images.githubusercontent.com/52473140/181070651-ab4aa15e-8e50-412c-9043-a09182a8d92b.png" alt="Screen Shot 2021-11-04 at 11.40.36 AM" data-base62-sha1="i2xdwF57EbTYGE32YVTqPcTYtkq" width="684" height="184"></p>
<p>So this is only an issue with the “live updates” of the pull-request. Let me know if you need any additional information.</p>

**Comment by solarmosaic-kflorence** [comment]
<p>I filed a support ticket for this here <a href="https://support.github.com/contact" class="inline-onebox" rel="noopener nofollow ugc">GitHub Support</a></p>

**Comment by solarmosaic-kflorence** [comment, marked as answer]
<p>Latest update from GitHub support:</p>
<blockquote>
<p>Our engineering team acknowledges the defect and working on it now. I cannot promise when this issue will be resolved, but please keep an eye on the <a href="https://github.blog/changelog/" rel="noopener nofollow ugc">Changelog</a> to stay up to date with new features, changes, and bug fixes.</p>
</blockquote>

**Comment by abelbraaksma** [comment]
I think this has been resolved, I don't see the duplicate lines anymore. However, there's still the issue of the enormous amount of clutter here, obscuring actual comments. See this discussion: https://github.com/community/community/discussions/20171.
